samples/testing/dump_operation/README.md

```diff
@@ -0,0 +1,37 @@
+**Copyright (c) Microsoft Corporation. All rights reserved.**  
+**Licensed under the MIT License.**
+
+# Testing Operations in Modern QDK
+This sample project demonstrates different approaches to testing operations in Modern QDK, which unlike Classic QDK, doesn't support native Q# tests.
```

Comment:
So far we've avoided specific references to the Classic QDK in our guidance here, since this QDK is expected to stand on it's own and not require folks to be familiar with the Classic QDK. I'd rephrase this as:
```suggestion
# Testing Operations in the QDK
This sample project demonstrates different approaches to testing operations in the QDK, both via Python and within Q# itself.
```

Reply:
I don't think that is needed, the guidance you have put together here should be enough for anyone searching for information on testing.